R/SEraster.R

```diff
@@ -0,0 +1,492 @@
+#' @include internal.R
+#'
+NULL
+#' createRasterizedObject
+#' @keyword internal
+#' 
+createRasterizedObject <- function(input, out, name) {
```

Comment:
Yep, thanks for reminding! - and `name` should probably be `new_assay_name` and/or be optional since I just realized `rasterizeCellType` passes in the metadata column name but it's never used (the condition under it is just to pull the right image to associate the assay with).